pytype/typegraph/cfg.cc

```diff
@@ -812,11 +812,7 @@ PyTypeObject PyCFGNode = {
 
 // --- Origin --------------------------------------------------------------
 
-static PyTypeObject PyOrigin = {
-  // initialized in inittypegraph
-  PyVarObject_HEAD_INIT(nullptr, 0)
-  tp_name: nullptr
-};
+static PyTypeObject PyOrigin;
```

Comment:
why this change?

Reply:
This is the problem explained in my initial comment:

> This was caused by a segmentation fault in cfg.cc. In Python 3.8, PyStructSequence_InitType was changed so that it errors out if a type with a reference count greater than 0 is passed in: https://github.com/python/cpython/blame/3.8/Objects/structseq.c#L396-L400
> 
> This error was being silently swallowed because we were just assuming that PyStructSequence_InitType was running successfully. The problem was we were using PyObject_HEAD_INIT which initializes the refcount to 1.

The recommended way to use PyStructSequence is this, as you can see in the stdlib:

* https://github.com/python/cpython/blob/578c3955e0222ec7b3146197467fbb0fcfae12fe/Objects/longobject.c#L5597
* https://github.com/python/cpython/blob/a487a39dca4c41305928c7dfdbcb0b3aa344683b/Python/thread.c#L164